js/copy_file_test.ts

```diff
@@ -31,6 +31,7 @@ testPerm({ write: true }, function copyFileSyncSuccess() {
   assertSameContent(fromFilename, toFilename);
 });
 
+/* Test is incorrect. TODO: fix this test.
```

Comment:
Would like to know what is wrong here too. I checked out this branch myself and seems this test is running fine (unlike the tests in `app_test.js`)

Reply:
Oh did not see branch commit history either. Checking out right now

Reply:
At least on windows, the error is ErrorKind.NotFound which is what I would hope and expect it to be.

Reply:
@piscisaureus `ErrorKind.NotFound` is from Rust's `std::fs::copy` (on mac, dunno if that is platform specific). I could definitely manually replace with our custom error

Reply:
So this is the error message on mac
```
InvalidInput: the source path is not an existing regular file
```
Not quite sure if this is behaving same on windows